tests/pfcwd/test_pfcwd_function.py

```diff
@@ -1105,8 +1126,17 @@ def test_pfcwd_port_toggle(self, request, fake_storm, setup_pfc_test, setup_dut_
                 reg_exp = loganalyzer.parse_regexp_file(src=ignore_file)
                 loganalyzer.ignore_regex.extend(reg_exp)
                 loganalyzer.expect_regex = []
-                loganalyzer.expect_regex.extend([EXPECT_PFC_WD_DETECT_RE + fetch_vendor_specific_diagnosis_re(duthost)])
-                loganalyzer.match_regex = []
+                if duthost.facts["asic_type"] in ['cisco-8000']:
+                    loganalyzer.expect_regex.extend(
```

Comment:
Can I understand you just remove the pfcwd storm detect/restore syslog checking here?